modules/services/media-ffmpeg/src/main/kotlin/au/com/shiftyjelly/pocketcasts/media/FFmpegMediaService.kt

```diff
@@ -120,12 +120,23 @@ internal class FFmpegMediaService(
         return executeAsyncCommand(command).map { outputFile }.getOrNull()
     }
 
-    private fun sanitizedFileName(podcast: Podcast, episode: PodcastEpisode, clipRange: Clip.Range): String {
-        return "${podcast.title} - ${episode.title} - ${clipRange.start.toSecondsWithSingleMilli()}–${clipRange.end.toSecondsWithSingleMilli()}".replace("""\W+""".toRegex(), "_")
-    }
-
-    private fun sanitizedFileName(podcast: Podcast, episode: PodcastEpisode, clipRange: Clip.Range, cardType: VisualCardType): String {
-        return "${podcast.title} - ${episode.title} - ${clipRange.start.toSecondsWithSingleMilli()}–${clipRange.end.toSecondsWithSingleMilli()}-${cardType.id}".replace("""\W+""".toRegex(), "_")
+    private fun sanitizedFileName(
+        podcast: Podcast,
+        episode: PodcastEpisode,
+        clipRange: Clip.Range,
+        cardType: VisualCardType?,
+    ) = buildString {
+        append(podcast.title.replace(sanitizedFileNameRegex, " "))
+        append(" - ")
+        append(episode.title.replace(sanitizedFileNameRegex, " "))
+        append(" - ")
+        append(clipRange.start.toSecondsWithSingleMilli())
+        append('-')
+        append(clipRange.end.toSecondsWithSingleMilli())
+        if (cardType != null) {
+            append('-')
+            append(cardType.id)
```

Comment:
The clip I exported had the following name.
`The World Today -  4 7 billion to tackle domestic violence - 0-15-v.mp4`
I wonder if it would be nicer to not shorten the card id add a space to the front. So instead, it would be:
`The World Today -  4 7 billion to tackle domestic violence - 0-15 - vertical.mp4`

Feel free to ignore this, just an idea.

Reply:
Both work for me. I'll update it